apps/chronicles/screens/dfw/dfwcomponents/WorkOrderCellScreen.tsx

```diff
@@ -4,8 +4,23 @@ import { WorkOrderCell } from "@equinor/mad-dfw";
 import { View } from "react-native";
 import { ScrollView } from "react-native-gesture-handler";
 
+type Operation = {
+    id: number;
+    description: string;
+};
 export const WorkOrderCellScreen = () => {
     const styles = useStyles(themeStyles);
+
+    const operations = [
+        { id: 1, description: "Inspection" },
+        { id: 2, description: "Maintenance" },
+    ];
+
+    const getOperationsText = (operations: Operation[]) => {
+        if (!operations || operations.length === 0) return "No operations";
+        if (operations.length === 1) return "1 operation";
```

Comment:
Usually in DFW(WorkOrder to be precise), if there is only one operation, then we show the `operationId` - `operationDescription` like the second workorder in this screenshot.

![image](https://github.com/equinor/mad/assets/75029767/a60af196-f7bc-4995-8054-187ce0fa918d)

Reply:
Is it possible to have it this way?